app/frontend/src/components/UserSummary.tsx

```diff
@@ -81,10 +81,25 @@ export default function UserSummary({
       {!user ? (
         <Skeleton />
       ) : (
-        <>{nameOnly ? user.name : `${user.name}, ${user.age}, ${user.city}`}</>
+        <>{nameOnly ? user.name : `${user.name}, ${user.age}`}
+        </>
       )}
     </Typography>
   );
+  const location = (
+    <Typography
+      color="textSecondary"
+      variant="body1"
+      noWrap={nameOnly}
+    >
+      {!user ? (
+        <Skeleton />
+      ) : (
+        <>{user.city}
+        </>
+      )}
+    </Typography>
+  )
```

Comment:
It doesn't look like this is being used elsewhere, so we can inline the JSX instead so the code is closer to where it's being used